components/Hero/Hero.jsx

```diff
@@ -0,0 +1,29 @@
+import styles from '@/styles/Hero.module.css'
+import Image from 'next/image'
+import banner from '../../public/parlement.png'
+import labels from '@/constants/labels.en'
+
+const Hero = () => {
+    return (
+        <div className={styles.container}>
```

Comment:
Hero image It's cutting out 

<img width="1791" alt="Screenshot 2023-03-06 at 6 27 12 PM" src="https://user-images.githubusercontent.com/11657599/223279935-0132a69a-090f-41d9-9925-6147e6a53de3.png">

Reply:
Mine is ok. Already zoomed in and out and the image is not being cut

![image](https://user-images.githubusercontent.com/57446278/223282272-2b2fbfd3-7357-47a0-ad41-a3bd16559625.png)

Reply:
Nor working on Safari. It could fixed later.

Reply:
I tested on firefox and chrome